packages/numbers/tests/rest/v1/available-number/available-number.steps.ts

```diff
@@ -0,0 +1,153 @@
+      regionCode: 'US',
+      type: 'LOCAL',
+      capabilities: ['SMS', 'VOICE'],
+      smsConfiguration: {
+        servicePlanId: 'SpaceMonkeySquadron',
+      },
+      voiceConfiguration: {
+        appId: 'sunshine-rain-drop-very-beautifulday',
+      },
+      numberPattern: {
+        pattern: '7654321',
+        searchPattern: 'END',
+      },
+    },
+  });
+});
+
+Then('the response contains an active phone number', () => {
+  assert.equal(activeNumber.phoneNumber, '+12017654321');
+  assert.equal(activeNumber.projectId, '123c0ffee-dada-beef-cafe-baadc0de5678');
+  assert.equal(activeNumber.displayName, '');
+  assert.equal(activeNumber.regionCode, 'US');
+  assert.equal(activeNumber.type, 'LOCAL');
+  assert.deepEqual(activeNumber.capability, ['SMS', 'VOICE']);
+  assert.deepEqual(activeNumber.money, { currencyCode: 'EUR', amount: '0.80' });
+  assert.equal(activeNumber.paymentIntervalMonths, 1);
+  assert.deepEqual(activeNumber.nextChargeDate, new Date('2024-06-06T14:42:42.022227Z'));
+  assert.equal(activeNumber.expireAt, null);
+  const expectedSmsConfiguration: Numbers.SMSConfiguration = {
+    servicePlanId: '',
+    campaignId: '',
+    scheduledProvisioning: {
+      servicePlanId: 'SpaceMonkeySquadron',
+      campaignId: '',
+      status: 'WAITING',
+      lastUpdatedTime: new Date('2024-06-06T14:42:42.596223Z'),
+      errorCodes: [],
+    },
+  };
+  assert.deepEqual(activeNumber.smsConfiguration, expectedSmsConfiguration);
+  const expectedVoiceConfiguration: Numbers.VoiceConfiguration = {
+    type: 'RTC',
+    appId: '',
+    trunkId: '',
+    serviceId: '',
+    lastUpdatedTime: null,
+    scheduledVoiceProvisioning: {
+      type: 'RTC',
+      appId: 'sunshine-rain-drop-very-beautifulday',
+      trunkId: '',
+      serviceId: '',
+      status: 'WAITING',
+      lastUpdatedTime: new Date('2024-06-06T14:42:42.604092Z'),
+    },
+  };
+  assert.deepEqual(activeNumber.voiceConfiguration, expectedVoiceConfiguration);
+  assert.equal(activeNumber.callbackUrl, '');
+});
+
+When('I send a request to rent the phone number {string}', async (phoneNumber: string) => {
+  activeNumber = await availableNumberApi.rent({
+    phoneNumber,
+    rentNumberRequestBody: {
+      smsConfiguration: {
+        servicePlanId: 'SpaceMonkeySquadron',
+      },
+      voiceConfiguration: {
+        appId: 'sunshine-rain-drop-very-beautifulday',
+      },
+    },
+  });
+});
+
+Then('the response contains this active phone number {string}', (phoneNumber: string) => {
+  assert.equal(activeNumber.phoneNumber, phoneNumber);
+});
+
+When('I send a request to rent the unavailable phone number {string}', async (phoneNumber: string) => {
+  activeNumber = await availableNumberApi.rent({
+    phoneNumber,
+    rentNumberRequestBody: {
+      smsConfiguration: {
+        servicePlanId: 'SpaceMonkeySquadron',
+      },
+      voiceConfiguration: {
+        appId: 'sunshine-rain-drop-very-beautifulday',
+      },
+    },
+  });
+});
```

Comment:
No `Then` defined/related to this `When`? (https://github.com/sinch/sinch-sdk-mockserver/pull/5/files#diff-9b479f26c322fe5bdd007abe50e50c80a0dc8e223ab7eb24fdc28591bb799bd8R33).

Reply:
The `Then` is on line 53